pkg/scheduler/factory_test.go

```diff
@@ -553,6 +554,53 @@ func TestDefaultErrorFunc_NodeNotFound(t *testing.T) {
 	}
 }
 
+func TestDefaultErrorFunc_PodAlreadyBound(t *testing.T) {
+	nodeFoo := &v1.Node{ObjectMeta: metav1.ObjectMeta{Name: "foo"}}
+	testPod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "test-pod", Namespace: "default"}, Spec: v1.PodSpec{NodeName: "foo"}}
+	tests := []struct {
+		name      string
+		nodes     []v1.Node
+		injectErr error
+	}{
+		{
+			name:      "pod has been bound during a scheduling cycle",
+			nodes:     []v1.Node{*nodeFoo},
+			injectErr: fmt.Errorf("binding rejected: timeout"),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			stopCh := make(chan struct{})
+			defer close(stopCh)
+
+			client := fake.NewSimpleClientset(&v1.PodList{Items: []v1.Pod{*testPod}}, &v1.NodeList{Items: tt.nodes})
+			informerFactory := informers.NewSharedInformerFactory(client, 0)
+			podInformer := informerFactory.Core().V1().Pods()
+			// Need to add testPod to the store.
+			_ = podInformer.Informer().GetStore().Add(testPod)
+
+			queue := internalqueue.NewPriorityQueue(nil, internalqueue.WithClock(clock.NewFakeClock(time.Now())))
+			schedulerCache := internalcache.New(30*time.Second, stopCh)
+
+			for i := range tt.nodes {
+				node := tt.nodes[i]
+				// Add node to schedulerCache no matter it's deleted in API server or not.
+				_ = schedulerCache.AddNode(&node)
+			}
+
+			testPodInfo := &framework.QueuedPodInfo{Pod: testPod}
+			errFunc := MakeDefaultErrorFunc(client, podInformer.Lister(), queue, schedulerCache)
+			errFunc(testPodInfo, tt.injectErr)
+
+			got := getPodFromPriorityQueue(queue, testPod)
+			if got != nil {
+				t.Fatalf("Unexpected pod:pod should not be in PriorityQueue when the NodeName of pod is not empty")
```

Comment:
show the pod name in the error message, e.g.
`t.Fatalf("Unexpected pod: %v...", pod.name)`

Reply:
I have fixed it